tvision/include/compatlayer.h

```diff
@@ -2113,16 +2110,15 @@ CLY_CFunc int  CLY_getcurdir(int drive, char *buffer);
  #define Include_sys_types 1
 #endif
 
-#if (defined(Include_sys_types) && !defined(Included_sys_types)) || defined(Force_CLY_ssize_t)
- #ifndef Included_sys_types
-   #define Included_sys_types 1
-   #include <sys/types.h>
- #endif
- /* Platforms where sys/types.h doesn't define ssize_t: */
- #if defined(Uses_CLY_ssize_t) && !defined(CLY_ssize_t)
-  #define CLY_ssize_t 1
-  typedef long ssize_t;
- #endif
```

Comment:
Force_CLY_ssize_t is not necessary.  The solution is to simply separate out the second if condition to make it independent of the first one.